cvehound/__main__.py

```diff
@@ -28,6 +28,7 @@ def check_config(config):
         'report',
         'all_files',
         'metadata',
+        'ignore',
```

Comment:
I think we can use already existing `--exclude` for this. Just check is it a CVE-id or file with CVE-ids.